projects/demo/src/app/components/bug-report/bug-report.component.ts

```diff
@@ -75,7 +75,9 @@ export class BugReportComponent implements OnInit {
 
   copy() {
     this.cb.copy(this.reportStr);
-    this.toaster.open('copied to clipboard!');
+    this.toaster.open('copied to clipboard!', "Dismiss", {
```

Comment:
minor change to comply with code style, these should be single-quotes

Reply:
I have no idea how to handle the workflow, but I added the change as an edit to the commit pull request.

Reply:
Not sure which conversation to tell you, but I removed your parentheses, does this still look good?